Site/SiteAnalyticsModule.php

```diff
@@ -362,6 +387,111 @@ public function getFacebookPixelCommandsInlineJavascript()
 	}
 
 	// }}}
+	// {{{ public function hasBingUET()
+
+	public function hasBingUET()
+	{
+		return (
+			$this->bing_uet_id != '' &&
+			!$this->analytics_opt_out
+		);
+	}
+
+	// }}}
+	// {{{ public function pushBingUETCommands()
+
+	public function pushBingUETCommands(array $commands)
+	{
+		foreach ($commands as $command) {
+			$this->bing_uet_commands[] = $command;
+		}
+	}
+
+	// }}}
+	// {{{ public function prependBingUETCommands()
+
+	public function prependBingUETCommands(array $commands)
+	{
+		foreach ($commands as $command) {
+			array_unshift($this->bing_uet_commands, $command);
```

Comment:
This works the same as prepend methods for Bing and Google, which are also not used.

It exists for google for our old segmenting code for A/B testing to ensure a command gets added to the start of the stack, and I copied it for Facebook and now Bing. I think it's worth keeping around only so that they all have similar commands available. If people strongly disagree I can drop the Facebook and Bing prepend methods.

I will however fix the order issue that you correctly noticed.